### DISCUSSION TITLE
Set up IIS to serve packages.

### DISCUSSION THREAD

**Discussion by chrisrestall** [Squirrel/Squirrel.Windows#1797] [General]
Is there any additional detail around setting up IIS for this?  I see the web.config piece but it's not clear what else needs to be done to serve the packages up in IIS?

**Comment by anaisbetts** [comment]
Any static file server will do, there is no special configuration to do for IIS.

**Reply by chrisrestall** [reply]
Thank you, wasn't sure on directory browsing and what not but have it up and running now..
